modules/brew/README.md

```diff
@@ -0,0 +1,136 @@
+
+**`tmpfiles.d homebrew.conf`:**
+- This configuration is telling SystemD to: automatically create these necessary directories on every system boot if not available & to give them permissions of the default user (UID 1000):
+  - `/var/lib/homebrew/`
+  - `/var/cache/homebrew/`
+  - `/home/linuxbrew/`
+
+**`brew-setup`:**
+- `brew-setup` SystemD service checks if main directory used by Brew exists (`/home/linuxbrew/.linuxbrew/`)  
+  & if `brew-setup` state file exists (`/etc/.linuxbrew`)
+- If one of those paths don't exist, than Homebrew tar is extracted from `/usr/share/homebrew/homebrew.tar.zst` to `/tmp/homebrew/`
+- Extracted Homebrew is than copied from `/tmp/homebrew/` to `/home/linuxbrew/` & permissions to it are set to default user (UID 1000)
+- Temporary directory `/tmp/homebrew/` is removed
+- Empty file `/etc/.linuxbrew` is created, which indicates that brew-setup (installation) is successful & which allows setup to run again on next boot when removed
+
+**Rest of the setup:**
+- `brew-update` is runnning in specified time to update Brew binary to latest version
+- `brew-upgrade` is runnning in specified time to upgrade Brew packages to latest versions
+
+## Configuration Options
+
+### `update-interval` (optional: string, default: '6h')
+Defines how often the Brew update service should run. The string is passed directly to `OnUnitInactiveSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `upgrade-interval` (optional: string, default: '8h')
+Defines how often the Brew upgrade service should run. The string is passed directly to `OnUnitInactiveSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `wait-after-boot-update` (optional: string, default: '10min')
+Time delay after system boot before the first Brew update runs. The string is passed directly to `OnBootSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `wait-after-boot-upgrade` (optional: string, default: '30min')
+Time delay after system boot before the first Brew package upgrade runs. The string is passed directly to `OnBootSec` in systemd timer. (Syntax: ['1d', '6h', '10m']).
+
+### `auto-update` (optional: boolean, default: true)
+If false, disables automatic activation of `brew-update.timer`.
+
+### `auto-upgrade` (optional: boolean, default: true)
+If false, disables automatic activation of `brew-upgrade.timer`.
+
+!!! warning
+    Please review the brew documentation carefully before modifying these settings.
+
+### `nofile-limits` (optional: boolean, default: false)
+Determines whether to apply nofile limits for Brew installations. When set to true, it increases the nofile limits to prevent certain packages from failing due to file limits. However, it's important to note that increasing nofile limits can have potential security implications. Defaults to false for security purposes.
+
+### `brew-analytics` (optional: boolean, default: true)
+Determines whether to opt-out of Brew analytics. When set to true, analytics are enabled.
```

Comment:
Is this in scope for the warning above? If not, consider moving it above the warning. IMO analytics opt-out is more user preference than something that should be warned about.

Reply:
Yes, I can move this.